app/docs/user-processes.md

```diff
@@ -0,0 +1,486 @@
+# User Processes
+
+This document describes the processes or use cases that users engage in when interacting with our POC prototype application.
+
+### Login Process
+
+> :warning: The login solution presented in this POC prototype is temporary and will be replaced with an existing authentication system such as KeyCloak.
+
+> **Who:** All User Personas
+> **When:** A registered user is not currently logged into the application.
```

Comment:
who and when are displayed in one line. it's annoying markdown requires additional empty line for a new line.

Reply:
Thanks, I'll fix it. It doesn't show in my preview plugin, so that's good to know.